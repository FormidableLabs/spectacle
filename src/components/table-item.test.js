import React from 'react';
import { mount } from 'enzyme';
import TableItem from './table-item';

describe('<TableItem />', () => {
  test('should render correctly', () => {
    const context = {
      styles: {
        components: {
          tableItem: {
            color: '#444'
          }
        }
      }
    };
    const wrapper = mount(<TableItem>Table Item Content</TableItem>, {
      context
    });
    expect(wrapper).toMatchSnapshot();
  });
});
