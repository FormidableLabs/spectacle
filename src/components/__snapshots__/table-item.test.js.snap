// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableItem /> should render correctly 1`] = `
<TableItem>
  <Styled(td)
    className={undefined}
    style={
      Array [
        Object {
          "color": "#444",
        },
        Object {},
        Object {},
        undefined,
      ]
    }
  >
    <td
      className="css-0"
      style={
        Array [
          Object {
            "color": "#444",
          },
          Object {},
          Object {},
          undefined,
        ]
      }
    >
      Table Item Content
    </td>
  </Styled(td)>
</TableItem>
`;
